src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -1,6 +1,7 @@
 const FEATURE_FLAG_NAMES = Object.freeze({
   facilityLocatorShowCommunityCares: 'facilityLocatorShowCommunityCares',
   profileShowDirectDeposit: 'profileShowDirectDeposit',
+  profileShowNotifications: 'profileShowNotifications',
```

Comment:
could you make this flag name a little more specific? "notifications" is pretty general. maybe something like "profileShowTextMessageNotifications" or "profileShowNotificationsViaTextMessage"?